CMBLensingSPT3GInterface/src/CMBLensingSPT3GInterface.jl

```diff
@@ -0,0 +1,223 @@
+module CMBLensingSPT3GInterface
```

Comment:
I wonder if there is a better name-- the `SPT3GI` part is a bit intense..

Reply:
CMBLensingSPT3GInterface was exactly what I would have named it, so I suppose thats a sign its the right thing.